src/main/java/frc/robot/poseestimation/posecalculation/PoseCalculator.java

```diff
@@ -16,7 +16,6 @@
 import edu.wpi.first.math.geometry.Twist2d;
```

Comment:
This is really old code of poseestimator. We're not using any of it by now. Since you'll probably merge this first, this is fine, but keep in mind that these changes will be deleted.